### PR TITLE
Remove scope prefix from filename

### PR DESCRIPTION
**Pull request by tiaanduplessis**
#21

### CODE REVIEW

**Comment by egoist** [comment]
Can you also update the tests regarding this fix?

File: src/get-rollup-options.js [hunk 1]
```diff
@@ -5,10 +5,14 @@ import req from 'req-cwd'
 import stringifyAuthor from 'stringify-author'
 
 function getDest(options, format, compress) {
-  const filename = options.filename
+  let filename = options.filename
   const dir = options.outDir
   let suffix = ''
 
+  if (filename.startsWith('@')) {
```
Comment (egoist) [inline src/get-rollup-options.js]: I think filename could contain `@` and does not have to be an scoped npm package name, like `@my-lib`. We may only need to update the logic of getting `filename` from `pkg.name`, I mean here https://github.com/egoist/bili/blob/master/src/get-config.js#L8

Comment (tiaanduplessis) [inline src/get-rollup-options.js]: Looks like a better option to me as well. Are you happy with checking for starting with '@' in [/src/get-config.js@`master`#L8](https://github.com/egoist/bili/blob/master/src/get-config.js#L8)?

Comment (egoist) [inline src/get-rollup-options.js]: Yep, that's ok to me 👍🏻

**Pull request merged**